src/components/calcite-link/calcite-link.tsx

```diff
@@ -75,21 +81,21 @@ export class CalciteLink {
       />
     );
 
-    const attributes = getAttributes(this.el, ["dir", "icon-end", "icon-start", "id"]);
     const Tag = this.childElType;
     const role = this.childElType === "span" ? "link" : null;
     const tabIndex = this.disabled ? -1 : this.childElType === "span" ? 0 : null;
 
     return (
       <Host role="presentation">
         <Tag
-          {...attributes}
           class={{ [CSS_UTILITY.rtl]: dir === "rtl" }}
           dir={dir}
           href={Tag === "a" && this.href}
           ref={this.storeTagRef}
+          rel={Tag === "a" && this.rel}
           role={role}
           tabIndex={tabIndex}
+          target={this.target}
```

Comment:
Should target only get passed to "a" tags?

Reply:
thanks, good catch!